src/main/java/github/weichware10/util/data/TrialData.java

```diff
@@ -3,13 +3,15 @@
 import github.weichware10.util.Enums;
 import github.weichware10.util.Enums.ToolType;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 import org.joda.time.DateTime;
 
 /**
  * Stores the TrialData for the different tools internally.
  *
- * <p>also used to transfer Data
+ * <p>
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocParagraphCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br><p> tag should be placed immediately before the first word, with no space after.